app/src/sharedTest/java/org/oppia/android/app/devoptions/ForceNetworkTypeFragmentTest.kt

```diff
@@ -0,0 +1,405 @@
+package org.oppia.android.app.devoptions
```

Comment:
This test should be in the same corresponding package as the component that it's testing. Ditto for all tests in this & future PRs.

Reply:
Refactored. PTAL. Thanks.